src/awsexplorer/defaultRegion.ts

```diff
@@ -34,12 +35,12 @@ enum OnDefaultRegionMissingOperation {
 }
 
 class DefaultRegionMissingPromptItems {
-    public static readonly add: string = localize('AWS.message.prompt.defaultRegionHidden.add', 'Yes')
+    public static readonly add: string = localizedText.yes
     public static readonly alwaysAdd: string = localize(
         'AWS.message.prompt.defaultRegionHidden.alwaysAdd',
         "Yes, and don't ask again"
     )
-    public static readonly ignore: string = localize('AWS.message.prompt.defaultRegionHidden.ignore', 'No')
+    public static readonly ignore: string = localizedText.no
```

Comment:
Nit: everywhere else, you're using `localizedText.yes` and `localizedText.no` directly instead of assigning it to a variable. Should that be done here as well?

Reply:
Here I wanted to be minimally invasive to the existing logic